Onboarding/Onboarding/Pages/ProfilePages/Certification.cs

```diff
@@ -12,187 +12,183 @@ namespace Onboarding.Pages.ProfilePages
 {
     public class Certification
     {
-        public void AddCertificate(IWebDriver driver, string certificate, string certificationFrom, string year)
+        //Define class with two objects
+        public IWebDriver driver;
+        Certification CertificationObj;
+
+        //Form constructor 1
+        public Certification(IWebDriver _driver)
         {
-            //Wait
-            string xpValue = "//div[@data-tab='fourth']//div[@class ='ui teal button ']";
-            WaitHelpers.WaitToBeClickable(driver, "XPath", xpValue, 5);
+            //initial driver object
+            this.driver = _driver;   
+        }
+
+        //Inintial object and link to constructor 1 by calling method
+        public void CertificationStepDefinitions()
```

Comment:
This I believe is not needed is there any reason you are doing this?

Reply:
Thanks. It's redundant.